### PR TITLE
Switch to hatchling commands and remove scripts

### PR DESCRIPTION
**Pull request by aminalaee**
Switch to Hatchling for package management.

Remove scripts directory and use Hatch commands.

### CODE REVIEW

**Comment by ofek** [comment]
Awesome use of scripts!!!

**Comment by aminalaee** [comment]
@ofek Thanks for the great work :)

File: pyproject.toml [hunk 1]
```diff
@@ -50,3 +51,59 @@ include = [
 exclude = [
   "tests/*",
 ]
+
+[tool.hatch.envs.default]
+dependencies = [
```
Comment (Kludex) [inline pyproject.toml:56]: Does dependabot work on those?

Comment (ofek) [reply pyproject.toml:56]: No

Comment (Kludex) [reply pyproject.toml:56]: this project uses dependabot, so I guess it should replace by https://github.com/renovatebot/renovate ?

Comment (ofek) [reply pyproject.toml:56]: seems stuck in the past too https://docs.renovatebot.com/configuration-options/#python

**Pull request merged**